lib/adapters/MochaAdapter.js

```diff
@@ -0,0 +1,112 @@
+import EventEmitter from 'events'
+import { Test, Suite } from '../Data.js'
+
+export default class MochaAdapter extends EventEmitter {
+  constructor (mocha) {
+    super()
+
+    this.mocha = mocha
+    mocha.reporter((runner) => {
```

Comment:
[Mocha Usage](https://mochajs.org/#usage), the --reporter description is `specify the reporter to use` and [Using Mocha programmatically](https://github.com/mochajs/mocha/wiki/Using-mocha-programmatically#set-options), `reporter` function description is `set the reporter`. 

So there is no option like `addReporter` or something like that :disappointed: